src/tool/subcommands/api_cmd.rs

```diff
@@ -1438,6 +1438,7 @@ fn eth_tests_with_tipset<DB: Blockstore>(store: &Arc<DB>, shared_tipset: &Tipset
             ))
             .unwrap(),
         ),
+        RpcTest::basic(EthGetBlockReceipts::request((block_hash.clone(),)).unwrap()),
```

Comment:
Any reason to use `basic` instead of `identity` here?

Reply:
I am actually a bit confused about when to use `basic` and when to use `identity` in this case

Reply:
Just to confirm, 
`basic`: Only checks schema validity
`identity`: Checks schema validity AND exact response equality

Reply:
That's correct!